UI/window-basic-settings.cpp

```diff
@@ -2006,17 +2002,21 @@ void OBSBasicSettings::LoadAudioDevices()
 
 void OBSBasicSettings::LoadAudioSources()
 {
+	if (ui->audioSourceLayout->rowCount() > 0) {
+		QLayoutItem *forDeletion = ui->audioSourceLayout->takeAt(0);
+		delete forDeletion->widget();
```

Comment:
OK, for Qt 5.5 compatibility on Linux, I've changed the contents of this conditional statement from `ui->audioSourceLayout->removeRow(0);`

I considered adding a compile-time check for the Qt version, but that'd require an extra import and I didn't feel it necessary as the code above seems to function without issue on newer versions of Qt too.